way_to_home/static/src/components/loginForm/loginForm.js

```diff
@@ -5,30 +5,36 @@ import Button from '@material-ui/core/Button';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import Checkbox from '@material-ui/core/Checkbox';
 import axios from 'axios';
+import CustomizedSnackbars from '../message/message';
 import './loginForm.css';
 
 
 class LoginForm extends Component {
 
     state = {
         request_type: 'login',
-        change_button: 'sign up',
+        confirm_button: 'увійти',
+        change_button: 'зареєструватися',
         repeat_display: 'none',
         email: '',
         first_pass: '',
         second_pass: '',
         email_error: false,
         pass_error: false,
         disable_button: true,
+        ajax_error: undefined,
```

Comment:
Rename this variable, cause in the future it may contain not only error from requests.

Reply:
For example?

Reply:
How your errors can get in my component?

Reply:
I make google authorization in this component

Reply:
But still it will be AJAX error

Reply:
ok, i will change by my hands)))

Reply:
but in react u should name ur var such ajaxError

Reply:
change all variables' name